### ISSUE TITLE
[Bug]: OCP\Files\NotFoundException: on file move

### ISSUE THREAD

**Issue by gleanlux** [nextcloud/server#36070]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When moving any file, an error occurs in the error log.
For each file moved, a new error entry is made.

No error when copying or deleting a file.

```
Hiba | no app in context | OCP\Files\NotFoundException: 
0. /var/www/html/cloud/lib/private/Files/Node/Node.php - line 355: 
OC\Files\Node\Node->getFileInfo()

1. /var/www/html/cloud/lib/private/Files/Node/Node.php - line 164: 
OC\Files\Node\Node->getMountPoint()

2. /var/www/html/cloud/apps/maps/lib/Hooks/FileHooks.php - line 188: 
OC\Files\Node\Node->getStorage()

3. /var/www/html/cloud/apps/maps/lib/Hooks/FileHooks.php - line 90: 
OCA\Maps\Hooks\FileHooks->isUserNode()

4. <<closure>>
OCA\Maps\Hooks\FileHooks->OCA\Maps\Hooks\{closure}("*** sensiti ... *")

5. /var/www/html/cloud/lib/private/Hooks/EmitterTrait.php - line 106: 
call_user_func_array()

6. /var/www/html/cloud/lib/private/Hooks/PublicEmitter.php - line 40: 
OC\Hooks\BasicEmitter->emit()

7. /var/www/html/cloud/lib/private/Files/Node/Root.php - line 143: 
OC\Hooks\PublicEmitter->emit()

8. <<closure>>
OC\Files\Node\Root->emit()

9. /var/www/html/cloud/lib/private/Files/Node/LazyFolder.php - line 72: 
call_user_func_array()

10. /var/www/html/cloud/lib/private/Files/Node/LazyFolder.php - line 100: 
OC\Files\Node\LazyFolder->__call()

11. /var/www/html/cloud/lib/private/Files/Node/HookConnector.php - line 190: 
OC\Files\Node\LazyFolder->emit()

12. /var/www/html/cloud/lib/private/legacy/OC_Hook.php - line 106: 
OC\Files\Node\HookConnector->postRename()

13. /var/www/html/cloud/lib/private/Files/View.php - line 860: 
OC_Hook::emit()

14. /var/www/html/cloud/apps/dav/lib/Connector/Sabre/Directory.php - line 442: 
OC\Files\View->rename()

15. /var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Tree.php - line 160: 
OCA\DAV\Connector\Sabre\Directory->moveInto()

16. /var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/CorePlugin.php - line 612: 
Sabre\DAV\Tree->move()

17. /var/www/html/cloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php - line 89: 
Sabre\DAV\CorePlugin->httpMove()

18. /var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 472: 
Sabre\DAV\Server->emit()

19. /var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 253: 
Sabre\DAV\Server->invokeMethod()

20. /var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 321: 
Sabre\DAV\Server->start()

21. /var/www/html/cloud/apps/dav/lib/Server.php - line 360: 
Sabre\DAV\Server->exec()

22. /var/www/html/cloud/apps/dav/appinfo/v2/remote.php - line 35: 
OCA\DAV\Server->exec()

23. /var/www/html/cloud/remote.php - line 171: 
require_once("/var/www/ht ... p")

```

### Steps to reproduce

1. Click any file menu, and "Move or copy".
2. Select Move.

### Expected behavior

The move is successful, but an error entry is made in the error log.

### Installation method

Community Manual installation with Archive

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "25.0.2.3",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 3
        },
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "theme": "",
        "loglevel": 0,
        "log_type": "file",
        "logfile": "data\/nextcloud.log",
        "logfilemode": 416,
        "logdateformat": "F d, Y H:i:s",
        "mysql.utf8mb4": true,
        "simpleSignUpLink.shown": false,
        "mail_sendmailmode": "smtp",
        "mail_smtpsecure": "tls",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "updater.release.channel": "stable",
        "default_phone_region": "HU"
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.17.0
  - admin_audit: 1.15.0
  - bruteforcesettings: 2.5.0
  - calendar: 4.2.1
  - camerarawpreviews: 0.8.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.0.2
  - contactsinteraction: 1.6.0
  - dav: 1.24.0
  - deck: 1.8.3
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - groupfolders: 13.1.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - maps: 0.2.4
  - nextcloud_announcements: 1.14.0
  - notes: 4.6.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - photos: 2.0.1
  - previewgenerator: 5.1.1
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - related_resources: 1.0.3
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - spreed: 15.0.2
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - tasks: 0.14.5
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - twofactor_totp: 7.0.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - circles: 0.14.2
  - dashboard: 7.0.0
  - encryption
  - files_external
  - news: 20.0.0
  - recommendations: 0.4.0
  - suspicious_login
  - user_ldap
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"qEXDUMWQomEvX3WTlJA5","level":3,"time":"January 10, 2023 14:21:55","remoteAddr":"***REMOVED SENSITIVE VALUE***","user":"Glean","app":"no app in context","method":"MOVE","url":"/cloud/remote.php/dav/files/Glean/Mobil%20Photos/2021/IMG_20220826_175257.jpg","message":"","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:108.0) Gecko/20100101 Firefox/108.0","version":"25.0.2.3","exception":{"Exception":"OCP\\Files\\NotFoundException","Message":"","Code":0,"Trace":[{"file":"/var/www/html/cloud/lib/private/Files/Node/Node.php","line":355,"function":"getFileInfo","class":"OC\\Files\\Node\\Node","type":"->"},{"file":"/var/www/html/cloud/lib/private/Files/Node/Node.php","line":164,"function":"getMountPoint","class":"OC\\Files\\Node\\Node","type":"->"},{"file":"/var/www/html/cloud/apps/maps/lib/Hooks/FileHooks.php","line":188,"function":"getStorage","class":"OC\\Files\\Node\\Node","type":"->"},{"file":"/var/www/html/cloud/apps/maps/lib/Hooks/FileHooks.php","line":90,"function":"isUserNode","class":"OCA\\Maps\\Hooks\\FileHooks","type":"->"},{"function":"OCA\\Maps\\Hooks\\{closure}","class":"OCA\\Maps\\Hooks\\FileHooks","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/cloud/lib/private/Hooks/EmitterTrait.php","line":106,"function":"call_user_func_array"},{"file":"/var/www/html/cloud/lib/private/Hooks/PublicEmitter.php","line":40,"function":"emit","class":"OC\\Hooks\\BasicEmitter","type":"->"},{"file":"/var/www/html/cloud/lib/private/Files/Node/Root.php","line":143,"function":"emit","class":"OC\\Hooks\\PublicEmitter","type":"->"},{"function":"emit","class":"OC\\Files\\Node\\Root","type":"->"},{"file":"/var/www/html/cloud/lib/private/Files/Node/LazyFolder.php","line":72,"function":"call_user_func_array"},{"file":"/var/www/html/cloud/lib/private/Files/Node/LazyFolder.php","line":100,"function":"__call","class":"OC\\Files\\Node\\LazyFolder","type":"->"},{"file":"/var/www/html/cloud/lib/private/Files/Node/HookConnector.php","line":190,"function":"emit","class":"OC\\Files\\Node\\LazyFolder","type":"->"},{"file":"/var/www/html/cloud/lib/private/legacy/OC_Hook.php","line":106,"function":"postRename","class":"OC\\Files\\Node\\HookConnector","type":"->"},{"file":"/var/www/html/cloud/lib/private/Files/View.php","line":860,"function":"emit","class":"OC_Hook","type":"::"},{"file":"/var/www/html/cloud/apps/dav/lib/Connector/Sabre/Directory.php","line":442,"function":"rename","class":"OC\\Files\\View","type":"->"},{"file":"/var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Tree.php","line":160,"function":"moveInto","class":"OCA\\DAV\\Connector\\Sabre\\Directory","type":"->"},{"file":"/var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":612,"function":"move","class":"Sabre\\DAV\\Tree","type":"->"},{"file":"/var/www/html/cloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpMove","class":"Sabre\\DAV\\CorePlugin","type":"->"},{"file":"/var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/html/cloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/html/cloud/apps/dav/lib/Server.php","line":360,"function":"exec","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/html/cloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->"},{"file":"/var/www/html/cloud/remote.php","line":171,"args":["/var/www/html/cloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/html/cloud/lib/private/Files/Node/Node.php","Line":110,"CustomMessage":"--"}}
```

**Comment by gleanlux** [comment]
> Hi, can you run `occ files:scan --all`and check if that fixes the issue?

I ran it, but unfortunately the error still persists.

**Comment by szaimen** [comment]
No idea then

**Comment by gleanlux** [comment]
> No idea then

Anyway, thanks, it was worth a try. :)
Any information you need, I will try to provide it to you at any time.

**Comment by gleanlux** [comment]
It's new situation:
I noticed that my log level was "0 Debug", so I changed it to 'loglevel' => 2, and restarted the server completely.
I don't know exactly if it was due to this or to the previous 'file scan' and restart, but at the moment the error is gone, no error entries are in the log.
As soon as I have time, I will change the log level back to 1, then to 0, and test.

**Comment by joshtrichards** [comment]
The `loglevel` itself shouldn't have made a difference: the trace you provided as an exception logged at level `3`. So maybe the restart or the scan.

What's the underlying filesystem? Oh, I just noticed:

> `       "function": "OCA\\Maps\\Hooks\\{closure}",`

The `maps` app is involved here. If you experience this problem again, see if disabling `maps` makes it impossible to reproduce. (This also may have been fixed at this point).
